tagging_end.md

```diff
@@ -0,0 +1,3 @@
+## Notes
+
+* In the Target list, a logical OR is applied on all tracks, a logical OR is applied on all chapters. Then a logical AND is applied between the Tracks list and the Chapters list to know if an element belongs to this Target.
```

Comment:
Wouldn't it be better to use the word "and" instead of a comma in this first sentence?

Reply:
This isn't new text but it the part that used to follow the metadata list in tagging.md. In the Make process now tagging.md the-xml-translation and tagging_end.md are concatenated. The pr overall does not intend any semantic change, so I suggest this is addressed in a subsequent pr.